File: src/main/java/net/kuryshev/webnotes/Task.java
package net.kuryshev.webnotes;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Data;

import javax.persistence.*;

@Data
@Entity
public class Task {
    @Id
    @SequenceGenerator(name = "task_seq", sequenceName = "task_seq", allocationSize = 1)
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "task_seq")
    private Long id;
    private String text;
    private String context;
    private Integer priority;
    private Integer timeEstimation;

    @JsonIgnore
    private String username;
}

File: src/main/java/net/kuryshev/webnotes/Task.kt
package net.kuryshev.webnotes

import com.fasterxml.jackson.annotation.JsonIgnore
import javax.persistence.*

@Entity
data class Task(
    @Id
    @SequenceGenerator(name = "task_seq", sequenceName = "task_seq", allocationSize = 1)
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "task_seq")
    var id: Long? = null,
    var text: String = "",
    var context: String = "",
    var priority: Int = 1,
    var timeEstimation: Int = 0,
    @JsonIgnore
    var username: String = ""
)